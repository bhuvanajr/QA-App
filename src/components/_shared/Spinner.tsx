import React from 'react';
import { Spinner as BSSpinner } from 'react-bootstrap';

export const Spinner: React.FC = () => {
  return (
    <BSSpinner animation="border" role="status">
      <span className="visually-hidden">Loading...</span>
    </BSSpinner>
  );
};
