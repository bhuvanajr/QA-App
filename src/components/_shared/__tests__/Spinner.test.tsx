import { render } from '@testing-library/react';
import React from 'react';
import { Spinner } from '../Spinner';

test('should render Spinner correctly', () => {
  const { container } = render(<Spinner />);
  expect(container).toMatchSnapshot();
});
