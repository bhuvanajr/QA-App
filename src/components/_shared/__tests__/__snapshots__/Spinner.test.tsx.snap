// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Spinner correctly 1`] = `
<div>
  <div
    class="spinner-border"
    role="status"
  >
    <span
      class="visually-hidden"
    >
      Loading...
    </span>
  </div>
</div>
`;
